routes/api/events.js

```diff
@@ -0,0 +1,94 @@
+const mongoose = require('mongoose');
+const express = require("express");
+const router = express.Router();
+const Event = require('../../models/Event');
+const validateEventInput = require('../../validations/event');
+const validateEventUpdate = require('../../validations/event');
+
+router.get("/allevents", (req, res) => {
```

Comment:
I tried GET localhost:5000/api/events/allevents on postman but this gives a 404 error